scan/utils.go

```diff
@@ -24,8 +24,93 @@ import (
 	"github.com/future-architect/vuls/config"
 	"github.com/future-architect/vuls/models"
 	"github.com/future-architect/vuls/util"
+	"github.com/k0kubun/pp"
+	"time"
 )
 
+func Wpscan() (err error) {
```

Comment:
__[Golint]__ exported function Wpscan should have comment or be unexported

<sub>posted by Sider</sub>

Reply:
implement